src/components/SchemaPropertyKeyValue.vue

```diff
@@ -6,7 +6,7 @@
   <template v-else>
     <p-key-value :label="property.title" :value="value" class="schema-property-key-value">
       <template v-if="isDefined && isJsonProperty" #value>
-        <CodeHighlighting language="json" :value="jsonValue" />
+        <CodeSnippet language="json" :snippet="jsonValue ?? ''" />
```

Comment:
not fully sold on using the null coalesce here. wasn't sure why the computed on line 33 didn't default to empty string or if there's even a good reason. I gotta go to bed now
```
const jsonValue = computed(() => stringifyUnknownJson(props.value) ?? undefined)
```